src/screens/explore-screen/tabs/ArtistsTab.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useCallback } from 'react'
+
+import { ParamListBase } from '@react-navigation/native'
+import { NativeStackNavigationProp } from '@react-navigation/native-stack'
+import { Button, StyleSheet, View } from 'react-native'
+
+import { useThemedStyles } from 'app/hooks/useThemedStyles'
+import { ThemeColors } from 'app/utils/theme'
+
+import { TabInfo } from './TabInfo'
+
+const messages = {
+  infoHeader: 'Featured Artists'
+}
+
+type Props = {
+  navigation: NativeStackNavigationProp<ParamListBase, keyof ParamListBase>
+}
+
+const createStyles = (themeColors: ThemeColors) =>
+  StyleSheet.create({
+    tabContainer: {
+      display: 'flex',
+      flexDirection: 'column'
```

Comment:
FYI the default flex direction in RN is `column`

Reply:
🤯